### DISCUSSION TITLE
Fokker–Planck simulation and fitting

### DISCUSSION THREAD

**Discussion by Jiashun97** [mwshinn/PyDDM#74] [Ask for help]
Could you provide a pointer to the code of Fokker–Planck simulation and fitting? Thank you.

**Comment by Jiashun97** [comment]
I just realize that the solver will be selected with a preference. So for a OU model, Crank-Nicolson is prefered, right?

**Comment by mwshinn** [comment]
PyDDM selects the solver automatically when you call "solve()".  You do not need to worry about what is being used under the hood. Right now, it actually calls backward Euler for almost everything by default because it has a C implementation whereas CN doesn't yet.  (But this should change soon...)  CN would be faster than BE if it had a C implementation, but it is still best not to use CN with collapsing bounds, so BE would be used then anyway.

**Reply by Jiashun97** [reply]
I see. Thanks for your answer.
